### DISCUSSION TITLE
SkyCulture Incomplete Loading

### DISCUSSION THREAD

**Discussion by PLANETARAS** [Stellarium/stellarium#3546] [General]
I often find some skyculture in the display options window on the newly loaded Stellarium 23.3. Every time they are different and you rarely find even one! How come? How can it be fixed?
![Senza titolo](https://github.com/Stellarium/stellarium/assets/97767104/455e7d82-24ce-4f73-a976-e6524bf67c93)

**Comment by gzotti** [comment]
"Western" has been renamed to "Modern" more than a year ago, strange to see the old name in 23.3. The logfile may provide hints on what went wrong. Else, did you install additional skycultures?

**Reply by PLANETARAS** [reply]
I've added previous skyculture by integrating folders

**Reply by gzotti** [reply]
The logfile may provide hints on what went wrong.

**Reply by 10110111** [reply]
I've seen a similar problem when I had a broken skyculture directory among the working ones: once a broken one is hit during the enumeration, the subsequent ones are not loaded either.
